### PR TITLE
Updated footer: Added the new responsive footer.

### PR DESCRIPTION
**Pull request by parv-FE-dev**

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            </div>
+          </div>
+          <div className="flex items-center 2xl:justify-center h-full pl-[30px] 2xl:pl-[0]">
+            <div className="grid grid-cols-4 gap-[15px] md:gap-[30px] lg:gap-[45px] 2xl:grid-cols-2 2xl:gap-[65px]">
+              <Link
+                href="https://www.youtube.com/channel/UCvFzTxpXy2fWmmP_Ge73Qhg"
+                target="_blank"
+              >
+                <Image
+                  src={youTubeLogo.src}
```
Comment (1rahul-1verma) [inline src/components/Footer/Footer.tsx:64]: we have used material icons, is this icon not present in it ?? @parv-FE-dev

File: src/components/Footer/Footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.footer_wrapper {
```
Comment (1rahul-1verma) [inline src/components/Footer/Footer.module.scss:1]: Why do we need to create this ?? we can use tailwind atomic classes instead to make it more understandable and scalable

File: src/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      .pseudo_block {
+        position: absolute;
+        background-color: #000;
+        right: 0;
+        bottom: -50px;
+        .pseudo_curve {
+          width: 50px;
+          height: 50px;
+          border-top-right-radius: 50px;
+          background-color: #1a1713;
```
Comment (1rahul-1verma) [inline src/components/Footer/Footer.module.scss:49]: This color is used at multiple places, can we just put these in out tailwind config and use it ??

Comment (1rahul-1verma) [reply src/components/Footer/Footer.module.scss:49]: please check `tailwind.config.js`

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+              <Link href="https://x.com/CapX_live" target="_blank">
+                <Image
+                  src={twitterLogo.src}
+                  alt="Twitter"
+                  width={72}
+                  height={72}
+                  className="cursor-pointer max-w-[30px] max-h-[30px] md:max-w-[45px] md:max-h-[45px] lg:max-w-[60px] lg:max-h-[60px] 2xl:max-w-[72px] 2xl:max-h-[72px]"
+                />
+              </Link>
+              <Link href="https://www.instagram.com/capx_live/" target="_blank">
```
Comment (1rahul-1verma) [inline src/components/Footer/Footer.tsx:80]: Next Link ?? we generally use this for linking to some page in our directory, as this provides optimisation by prefetching the page. Do we need it for external links as well ?? We might be overkilling here by using Link

File: src/components/Footer/Footer.module.scss (hunk 2 above)
Comment (1rahul-1verma) [inline src/components/Footer/Footer.module.scss:1]: incorrect folder name `Footer`.
folder name should start with small case. and while creating a folder, we should add an index file in the folder and export our component from there.
Please check this for ref `src/components/featureSlider/index.ts`

Comment (1rahul-1verma) [inline src/components/Footer/Footer.module.scss:1]: It is showing 
image
 to me while local build.
 Next doesn't support scss file or css file.
 we should use tailwind atomic classes or inline css if requried

**Review by 1rahul-1verma — COMMENTED** [review]
Please only commit required file.
which are under pages, public, src/component directory.

**Comment by parv-FE-dev** [comment]
> Please only commit required file. which are under pages, public, src/component directory.

**Pull request closed without merging**